packages/api/src/handlers/habitat-site/post-habitat-site.js

```diff
@@ -27,8 +28,9 @@ export default async (context, req, h) => {
         .code(409)
     }
 
+    req.payload.id = id
     const { dataValues } = await models.habitatSites.create({
-      id: uuidv4(),
```

Comment:
On a POST the key is generated as the RESTful convention is that POST creates an item where as PUT updates it. The id value should never be submitted as part of the payload. (Even though there is an OpenAPI problem which displays the Id in the example payload)

Reply:
![Screenshot 2022-09-21 at 09 23 07](https://user-images.githubusercontent.com/42332389/191454011-c03e6523-aa57-4b8e-a75e-b7dad8951eff.png)


The reason is that we did this, the id at the top level and the id in the payload were different, and we could not retrieve data from the API due to the mismatch.